cmd/build.go

```diff
@@ -0,0 +1,197 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/containerd/console"
+	"github.com/moby/buildkit/client"
+	"github.com/moby/buildkit/session"
+	"github.com/moby/buildkit/session/auth/authprovider"
+	"github.com/moby/buildkit/util/progress/progressui"
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/k8s/forward"
+	"github.com/okteto/okteto/pkg/k8s/namespaces"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	frontend          = "dockerfile.v0"
+	buildKitContainer = "buildkit-0"
+	buildKitPort      = 1234
+)
+
+//Build build and optionally push a Docker image
+func Build() *cobra.Command {
+	var file string
+	var tag string
+	var target string
+	var noCache bool
+
+	cmd := &cobra.Command{
+		Use:   "build",
+		Short: "Build (and optionally push) a Docker image",
```

Comment:
should we say Docker even if we are using `Buildkit`?

Reply:
it is a docker image what we build. buildkit supports other formats